src/opdsrequestmanager.cpp

```diff
@@ -21,6 +26,44 @@ int OpdsRequestManager::getCatalogPort()
          : 443;
 }
 
+// Helper to handle replies and follow redirects
+void OpdsRequestManager::handleReply(QNetworkReply* reply, std::function<void(QNetworkReply*)> finalHandler, int redirectCount) {
+    QVariant redirectAttr = reply->attribute(QNetworkRequest::RedirectionTargetAttribute);
+    if (redirectAttr.isValid() && redirectCount < kMaxRedirects) {
+        QUrl redirectUrl = redirectAttr.toUrl();
+        // Make absolute if needed
+        if (redirectUrl.isRelative()) {
+            redirectUrl = reply->url().resolved(redirectUrl);
+        }
+        qInfo() << "Following redirect" << redirectCount + 1 << "to:" << redirectUrl.toString();
```

Comment:
It would be nice to have the URL that was redirected (`reply->url()`) in this info message.